slurm_pipe.py

```diff
@@ -54,7 +48,10 @@
     config_params = pickle.load(fp)
 fp.close()
 
-logger=setup_logging(config_params["output_path"],True,False)
+logger = setup_logging(
```

Comment:
Approval of this change will depend on review of the modified setup_logging function - marking for future review.

Reply:
Yep, I'm all good here.